Main.py

```diff
@@ -9,7 +9,12 @@
 from ExtractMainContent import ExtractMainContent
 import sys
 import codecs
+import signal
 
+def signal_handler(signal, frame):

```

Comment:
can you add a comment for this and its purpose?

It is good to maintain a uniform indentation.